### PR TITLE
Improve test coverage of WorkspaceDocument, WorkspacePackageRepository and ExtendedWorkspaceDocumentManagerImpl

### PR DESCRIPTION
**Pull request by rasika**

### CODE REVIEW

File: language-server/modules/langserver-compiler/src/test/java/org/ballerinalang/langserver/compiler/workspace/repository/WorkspacePackageRepositoryTest.java [hunk 1]
```diff
@@ -0,0 +1,190 @@
...
+import org.wso2.ballerinalang.compiler.util.CompilerContext;
+import org.wso2.ballerinalang.compiler.util.Name;
+import org.wso2.ballerinalang.compiler.util.Names;
+import org.wso2.ballerinalang.compiler.util.diagnotic.BLangDiagnosticLog;
+
+import java.nio.file.Path;
+import java.nio.file.Paths;
+
+/**
+ * Tests for WorkspacePackageRepository.
```
Comment (nadeeshaan) [inline language-server/modules/langserver-compiler/src/test/java/org/ballerinalang/langserver/compiler/workspace/repository/WorkspacePackageRepositoryTest.java]: Let's add the version

File: language-server/modules/langserver-compiler/src/test/java/org/ballerinalang/langserver/compiler/workspace/WorkspaceDocumentTest.java [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.Test;
+
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+
+/**
+ * Tests for WorkspaceDocument.
```
Comment (nadeeshaan) [inline language-server/modules/langserver-compiler/src/test/java/org/ballerinalang/langserver/compiler/workspace/WorkspaceDocumentTest.java]: Let's add the version

Comment (rasika) [reply language-server/modules/langserver-compiler/src/test/java/org/ballerinalang/langserver/compiler/workspace/WorkspaceDocumentTest.java]: Fixed in [ecaf52d](https://github.com/ballerina-platform/ballerina-lang/pull/10194/commits/ecaf52d98daa0a19f68c882272fbae7284876b24)

File: language-server/modules/langserver-compiler/src/test/java/org/ballerinalang/langserver/compiler/workspace/repository/WorkspacePackageRepositoryTest.java (hunk 1 above)
Comment (rasika) [reply language-server/modules/langserver-compiler/src/test/java/org/ballerinalang/langserver/compiler/workspace/repository/WorkspacePackageRepositoryTest.java]: Fixed in [ecaf52d](https://github.com/ballerina-platform/ballerina-lang/pull/10194/commits/ecaf52d98daa0a19f68c882272fbae7284876b24)

**Pull request merged**